spec/users_controller_spec.rb

```diff
@@ -0,0 +1,34 @@
+require_relative 'rails_helper'
+RSpec.describe 'UsersController', type: :request do
+  describe 'GET #index' do
+    it 'returns http success' do
+      # this will perform a GET request to the /users route
+      get '/users'
+
+      # Check the http response for the correct placeholder text
+      # expect(response.body).to include('User Index Works')
+
+      # Check the http status of the response
+      expect(response.status).to eq(200)
+
+      # Check if the correct template was rendered
+      expect(response).to render_template('index')
+    end
```

Comment:
- [ ] I would also like to see the tests that ensure the response body includes the correct placeholder for all the specs.

ex: `expect(response.body).to include("User index Works")`

Reply:
Hello @Deepakdanger sure that is possible let me update and push, am sorry for that, I actually added it but commented the lines after.